### ISSUE TITLE
build an HTML example

### ISSUE THREAD

**Issue by jonotron** [jonotron/chromahash#4]
It would be helpful to have an example in HTML where users could try out the color hashing to see how it works for themselves

**Comment by Dragonfore** [comment]
Hello Jonathan, I'd like to work on an html example and am wondering if you'd like to use bootstrap to add styling or simply just an html page?

**Comment by jonotron** [comment]
I'm in favour of the simplest way possible with the least amount of external dependencies. A simple html file with some very simple styles would be sufficient. But really, anything that looks servicable would be great! If you want to use something like bootstrap, fly at it... I only ask that you try to source things from CDNs rather than putting them in the repository.

Hopefully the hash function is easy to pull into the browser. If you run into trouble doing that I can help you out.

Thanks a bunch!

**Comment by Dragonfore** [comment]
Alright sounds good. Sourcing from a CDN is surely the way to go, and I am not too worried about having it imported but will let you know if I have any issues. Thanks!

**Comment by jonotron** [comment]
@Dragonfore if you are still interested in this, I've added a web bundle of chromahash which you can see working (via console) in [`index.html`](https://github.com/jonotron/chromahash/blob/master/index.html)

**Comment by Dragonfore** [comment]
@jonotron Yes! It has been in the back of my mind since Hacktoberfest, and I would like to continue to work on it! Primarily I have been focussing on school until things calm down, then get back into the roll of things, thanks for uploading that as well ^.^
